src/main/java/carpetaddonsnotfound/lists/BlockList.java

```diff
@@ -127,6 +139,7 @@ public class BlockList {
           Blocks.STRIPPED_CRIMSON_HYPHAE,
           Blocks.STRIPPED_WARPED_HYPHAE,
           Blocks.STRIPPED_MANGROVE_WOOD,
+          Blocks.STRIPPED_CHERRY_WOOD,
```

Comment:
Are you sure that Stripped Bamboo Wood isn't a thing?

Reply:
Just stripped bamboo block. Check [here](https://minecraft.fandom.com/wiki/Java_Edition_1.20#Additions).

The idea of a "Wood" doesn't exist for Bamboo, it is Mosaic instead